docs/tutorials/training_agents/FrozenLake_tuto.py

```diff
@@ -0,0 +1,564 @@
+"""
+Frozenlake benchmark
+====================
+
+"""
+
+
+# %%
+# In this post we'll compare a bunch of different map sizes on the
+# `FrozenLake <https://gymnasium.farama.org/environments/toy_text/frozen_lake/>`__
+# environment from the reinforcement learning
+# `Gymnasium <https://gymnasium.farama.org/>`__ using the Q-learning
+# algorithm.
+#
+
+
+# %%
+# Dependencies
+# ------------
+#
+
+
+# %%
+# Let's first import a few dependencies we'll need.
+#
+
+from collections import namedtuple
+from pathlib import Path
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import seaborn as sns
+from tqdm import tqdm
+
+import gymnasium as gym
+from gymnasium.envs.toy_text.frozen_lake import generate_random_map
+
+
+sns.set_theme()
+
+# %load_ext lab_black
+
+
+# %%
+# Parameters we'll use
+# --------------------
+#
+
+Params = namedtuple(
+    "Params",
+    [
+        "total_episodes",
+        "learning_rate",
+        "gamma",
+        "epsilon",
+        "map_size",
+        "seed",
+        "is_slippery",
+        "n_runs",
+        "action_size",
+        "state_size",
+        "proba_frozen",
+        "savefig_folder",
+    ],
+)
+p = Params(
```

Comment:
Could you rename to `params` to so it is easier to remember compared to `p`

Reply:
Done in https://github.com/Farama-Foundation/Gymnasium/pull/376/commits/21e494fee066dcee2f79207e13d4b6bc82cfb4ea.